CHANGELOG.develop

```diff
@@ -1505,6 +1539,8 @@ Benner and Paweł Siudak):
 - Restore =auto-completion-idle-delay= value for =company-idle-delay= (thanks to
   Miciah Dashiel Butler Masters)
 - Added =shell-default-width= layer variable (thanks to David Balatero)
+- Do not adjust the width of a popup shell's window unless its position is left
+  or right (thanks to Miciah Masters)
```

Comment:
This is a fix to the change described by the entry that immediately precedes it, so you can just add the name to the "thanks to" of the previous entry.